src/main/java/use_case/home/HomeOutputData.java

```diff
@@ -0,0 +1,13 @@
+package use_case.home;
+
+import java.util.ArrayList;
+
+public class HomeOutputData {
+    private ArrayList<String> wantedPosts;
+    public HomeOutputData(ArrayList<String> wantedPosts){
```

Comment:
We might have to make this (and any references to it) an ArrayList<Item> so we have the ID data required to make the Home page interactive (when we get there).